### PR TITLE
update: adding wizard to Wasm Get started tutorial

### PR DESCRIPTION
**Pull request by AlejandraPedroza**
Added wizard to the first part of the Get started tutorial, changed screenshots, and minor text updates.

### CODE REVIEW

**Review by JSMonk — APPROVED** [review]

File: docs/topics/wasm/wasm-get-started.md [hunk 1]
```diff
@@ -9,33 +9,38 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
...
-   git clone git@github.com:Kotlin/kotlin-wasm-examples.git
-   ```
-   
-> Alternatively, you can use our experimental [web wizard](https://kmp.jetbrains.com/) to download a sample project.
+> The Kotlin Multiplatform wizard is an [Experimental](components-stability.md) feature. It may be changed at any time.
 >
 {type="note"}
 
+1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
+2. On the **New Project** tab, change the project name to "ComposeDemo" and the project ID to "kmp.project.demo".
```
Comment (zamulla) [inline docs/topics/wasm/wasm-get-started.md:19]: If we're guiding the user toward a purely Wasm project, the names here could reflect that. But it's just a suggestion :)

File: docs/topics/wasm/wasm-get-started.md [hunk 2]
```diff
@@ -9,33 +9,38 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
...
+## Open the project in IntelliJ IDEA
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
+3. Navigate to the unpacked "ComposeDemo" folder and click **Open**.
+
 ## Run the application
 
-1. Open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+1. IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
```
Comment (zamulla) [inline docs/topics/wasm/wasm-get-started.md:33]:
```suggestion
1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
```

File: docs/topics/wasm/wasm-get-started.md [hunk 3]
```diff
@@ -9,33 +9,38 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
...
    {type="note"}
 
-2. In **compose-example** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
+2. In **composeApp** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
 
-   ![Run the Gradle task](wasm-gradle-task-window.png){width=650}
+   ![Run the Gradle task](wasm-gradle-task-window.png){width=600}
 
-    Alternatively, you can run the following command in the terminal from the `compose-example` directory:
+   Alternatively, you can also run the following command in the terminal from the `composeApp` directory:
```
Comment (zamulla) [inline docs/topics/wasm/wasm-get-started.md:43]: "Alternatively" doesn't need an "also," I think. It's already an alternative :)

**Review by zamulla — CHANGES_REQUESTED** [review]
Nice job!

Several things to note, in the comments.

File: docs/topics/wasm/wasm-get-started.md [hunk 4]
```diff
@@ -44,24 +49,29 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
...
+   http://localhost:8080/
    ```
 
-   You see a "Hello World!" button. Click it:
+   >The port number can vary because the 8080 port may be unavailable. You can find the actual port number printed
+   > in the Gradle build console.
+   >
+   {type="tip"}
+
+   You see a "Click me!" button. Click it:
```
[the remark this answers is not in the document]
Comment (AlejandraPedroza) [reply docs/topics/wasm/wasm-get-started.md:60]: I'm changing it to  "You can see a "Click me!" button" to keep it in the present tense. What do you think?

**Review by zamulla — APPROVED** [review]
LGTM!

**Pull request merged**